client/app/components/conversation.coffee

```diff
@@ -146,6 +146,9 @@ module.exports = React.createClass
                 messages.push(last = []) unless _.isArray(last)
                 last.push key
 
+        closeURL = Router.buildURL
```

Comment:
Note : having this closeURL value not be part of state or props will cause the following bug :
- I open message list & conversation -> both are rendered, closeURL is correct
- I change the filter in message-list -> message-list is rendered but NOT conversation, closeURL doesn't change
- I click the close button -> I go back to message-list without filter

The closeURL should instead belong to AppStateStore and be fetched in getStateFromStore